proposals/governance/incubation_dd.md

```diff
@@ -0,0 +1,60 @@
+# KubeVirt Incubation Stage Review Proposal
+
+## What is KubeVirt: Refresh
+KubeVirt technology addresses the needs of development teams that have adopted or want to adopt Kubernetes but possess existing Virtual Machine-based workloads that cannot be easily containerized. More specifically, the technology provides a unified development platform where developers can build, modify, and deploy applications residing in both Application Containers as well as Virtual Machines in a common, shared environment.
+
+Benefits are broad and significant. Teams with a reliance on existing virtual machine-based workloads are empowered to rapidly containerize applications. With virtualized workloads placed directly in development workflows, teams can decompose them over time while still leveraging remaining virtualized components as is comfortably desired.
+
+## Statement on alignment with the CNCF mission:
+
+## Project progress since sandbox
+
+## Metrics
+
+### Mailing list
+KubeVirt currently (2021/06/01) has 436 subscribers.  The mailing list typically get 0-5 new threads per day.
+
+### GitHub
+* 171 official members in the organization
+* About 50 contributors measured by multiple L or greater sized contributions
+* 96 watched tags
+* 2562 stars
+* 615 forks
+
+### Slack
+Slack channels typically get 0-5 new threads per day.
+
+### Downloads
+
+### Integrations
+* Kubernetes / minikube
+* Red Hat OKD / OpenShift 4.x
+* oVirt / Red Hat Virtualization
+
```

Comment:
Rancher and Kubermatic advertise the use of KubeVirt.  

Can you confirm Gardener?  Gardener looks like something else.  I even checked their go vendors directory in GitHub.  To me it doesn't look like they source Kubevirt.

Reply:
KubeVirt is a supported infrastructure provider for Project Gardner
https://github.com/gardener/gardener/tree/master/extensions

Reply:
BTW, Apple engineers have made it clear that they have a very similar use case for KubeVirt (deploying lightweight k8s clusters on top of it), which they try to open-source. We may want to mention their discussion of [cluster-api-provider-kubevirt plans](https://docs.google.com/document/d/1Gmc7LyCIL_148a9Tft7pdhdee0NBHdOfHS1SAF0duI4/edit#bookmark=id.7qrjiv2e5ho0) in [SIG Cluster Lifecycle](https://www.youtube.com/watch?v=3XwXbrM2Kxg)

Reply:
@dankenigsberg  Indeed.  Approval is being worked on through the OSPO / CNCF interface.  We have to tread carefully on the topic.

Reply:
@peterclauterbach Thanks for that link.  Do you have a contact at Gardener whom we can reach out to for corp approval?  Can we just add them since KubeVirt is listed on their public repo.

Reply:
Per OSPO KubeVirt has enough documented users to continue with the process.  Apple has a seperate issue to obtain permission. ([issue](https://github.com/kubevirt/kubevirt.github.io/issues/782)).  Gardener has a seperate issue to obtain permission. ([issue](https://github.com/kubevirt/kubevirt.github.io/issues/793)).  There are others  that I just found out about also.  I will reach out to them all individually as well.

I'm resolving this conversation.